soh/src/code/z_onepointdemo.c

```diff
@@ -69,7 +69,30 @@ s32 OnePointCutscene_SetInfo(PlayState* play, s16 camIdx, s16 csId, Actor* actor
     PosRot sp8C;
     f32 tempRand;
     Unique9OnePointCs* csInfo = ONEPOINT_CS_INFO(csCam);
+    
+    //the default is 90, lower values necessary to prevent camera swing as animation speeds up
+    //arrived at via testing camera movement when holding left/right during the animation
+    s16 camCrawlTemp = CVarGetInteger("gCrawlSpeed", 0);
+    s16 camCrawlTimer;
```

Comment:
I recommend initializing this variable just because that's a good programming practice.  Since the original default is 90, you'd might as well initialize to 90 here even though you also set that in the switch below.

Reply:
Done!